platform/bpl/linux/bpl_arp.cpp

```diff
@@ -0,0 +1,55 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Intel Corporation
```

Comment:
I know I'm nitpicking, but the year should be just 2019 here.

Reply:
right, will be changed